pkgs/development/tools/dot-http/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, rustPlatform, fetchFromGitHub, openssl, pkg-config, lib, libiconv, darwin }:
```

```suggestion
{ stdenv, rustPlatform, fetchFromGitHub, pkg-config, openssl, libiconv, Security }:
```

`lib` is not needed (see below), it is somewhat of a convention to order native build inputs before build inputs.